feconf.py

```diff
@@ -20,29 +20,15 @@
 import datetime
 import os
 
+from constants import constants  # pylint: disable=relative-import
+
 # Whether to unconditionally log info messages.
 DEBUG = False
 
-# The platform for the storage backend. This is used in the model-switching
-# code in core/platform.
-PLATFORM = 'gae'
-
-# This variable is for serving minified resources
-# when set to True. It reflects we are emulating running Oppia in a production
-# environment.
-FORCE_PROD_MODE = False
-
-# Whether we should serve the development or production experience.
-# DEV_MODE should only be changed to False in the production environment,
-# or if you want to use minified resources in the development environment.
-
-if PLATFORM == 'gae':
-    DEV_MODE = (
-        (not os.environ.get('SERVER_SOFTWARE') or
-         os.environ['SERVER_SOFTWARE'].startswith('Development')) and
-        not FORCE_PROD_MODE)
-else:
-    raise Exception('Invalid platform: expected one of [\'gae\']')
+# When DEV_MODE = true check that we are running in development enviroment.
+if (constants.DEV_MODE and os.environ.get('SERVER_SOFTWARE') and
```

Comment:
I think maybe we can drop the "and os.environ.get('SERVER_SOFTWARE')" bit? That way, a KeyError will result if that env var is not present for any reason, and we won't shadow it with that check.

Reply:
Done.